modules/virt-accessing-vmi-ssh.adoc

```diff
@@ -4,51 +4,83 @@
 
 :_content-type: PROCEDURE
 [id="virt-accessing-vmi-ssh_{context}"]
-= Accessing a virtual machine instance via SSH
+= Accessing a virtual machine via SSH by using virtctl
 
-You can use SSH to access a virtual machine (VM) after you expose port 22 on it.
+You can use the `virtctl ssh` command to forward SSH traffic to a virtual machine (VM) by using your local SSH client.
 
-The `virtctl expose` command forwards a virtual machine instance (VMI) port to a node
-port and creates a service for enabled access. The following example creates
-the `fedora-vm-ssh` service that forwards traffic from a specific port of cluster nodes to port 22 of the `<fedora-vm>` virtual
-machine.
+[NOTE]
+====
+Heavy SSH traffic on the control plane can slow down the API server. If you regularly need a high amount of connections and traffic, use a dedicated Kubernetes `Service` object to access the virtual machine.
+====
 
 .Prerequisites
-* You must be in the same project as the VMI.
```

Comment:
I just noticed this still applies. Either the user has to be in the same project or the different namespace/project needs to be specified.